src/parameters/parameters_flow_solver.cpp

```diff
@@ -94,6 +96,7 @@ void FlowSolverParam::parse_parameters(dealii::ParameterHandler &prm)
         else if (flow_case_type_string == "burgers_viscous_snapshot")  {flow_case_type = burgers_viscous_snapshot;}
         else if (flow_case_type_string == "burgers_rewienski_snapshot")  {flow_case_type = burgers_rewienski_snapshot;}
         else if (flow_case_type_string == "naca0012")  {flow_case_type = naca0012;}
+        else if (flow_case_type_string == "advection_explicit_time_study")  {flow_case_type = advection_explicit_time_study;}
```

Comment:
Time study should not be a flow type, `advection` makes more sense, the purpose for `flow_case` is to only simulate the flow (integrate to the final time). The integration test `.cpp` file takes care of the `time_study` component, and the parameter file `.prm` and `ctest` name takes care of the `explicit` component

Reply:
Thanks for the naming suggestion, I wasn't sure that this was the best choice! I would prefer to change the name to `advection_periodic` so that it isn't identical to the `pde_type`. I think this choice still reflects the contents of the `flow_case`, rather than the test that I'm running.